clinica/pipelines/cli_param/option.py

```diff
@@ -24,8 +24,10 @@
 
 dont_save_warped_unmodulated = option(
     "-dswu",
-    "--dont_save_warped_unmodulated",
-    is_flag=True,
+    "--dont_save_warped_unmodulated/--save_warped_unmodulated",
+    "dont_save_warped_unmodulated",
+    default=False,
```

Comment:
does `default=False` here means: `save_warped_unmodulated`?  Regardless, I think that for consistency we ought to have the non-negated action first (i.e save_warped_unmodulated/dont_save_warped_unmodulated). What do you think?

Reply:
> does default=False here means: save_warped_unmodulated?

Yes

> I think that for consistency we ought to have the non-negated action first. What do you think?

What matters is the logic of the target variable. Here the code expects a boolean with the behaviour of `dont_save_warped_unmodulated`, not its complement. If we swap the values as you propose (so essentially turning `dont_save_warped_unmodulated` into `save_warped_unmodulated`), then we'd also need to complement the actual value everywhere this boolean is used in code.

In the end, it's nitpicking and I don't particularly care either or tbh. Your call.

Reply:
All right, fair point. I guess we can leave it as is for now.

Reply:
Actually, on second thought and after seeing in the code that we end up doing `"save_warped_unmodulated": not dont_save_warped_unmodulated` maybe it is better to have be consistent with the non-negated part first. 
What do you think @mdiazmel  ?

Reply:
Assuming it's the only location where this boolean is used, then we'd be better off avoiding the double negation indeed.

Reply:
> Actually, on second thought and after seeing in the code that we end up doing "save_warped_unmodulated": not dont_save_warped_unmodulated maybe it is better to have be consistent with the non-negated part first.
What do you think @mdiazmel ?

I think that the best option is to keep the original behavior everywhere is possible.